database/migrations/2023_06_15_210220_create_user_logs_table.php

```diff
@@ -1,37 +1,32 @@
 <?php
 
-use Illuminate\Support\Facades\Schema;
-use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
 
-class CreateUserLog extends Migration
+return new class extends Migration
 {
     /**
      * Run the migrations.
-     *
-     * @return void
      */
-    public function up()
+    public function up(): void
     {
-        Schema::create('user_logs', function (Blueprint $table) {
-            $table->increments('id');
+        Schema::connection('logs')->create('user_logs', function (Blueprint $table) {
+            $table->id();
             $table->integer('user_id')->unsigned();
             $table->unsignedTinyInteger('type_id')
                 ->default(\App\Models\UserLog::TYPE_LOGIN);
             $table->string('ip', 255)->nullable();
+            $table->char('country', 6)->nullable();
             $table->timestamps();
-
```

Comment:
Same here, since we're recreating the user_logs table, might as well add the index for the purging of old rows